pkg/sriov/sriov.go

```diff
@@ -183,7 +198,33 @@ func (s *sriovManager) ReleaseVF(conf *sriovtypes.NetConf, podifName string, net
 	if err != nil {
 		return fmt.Errorf("failed to get init netns: %v", err)
 	}
-
+	tempNS, err := ns.TempNetNS()
+	if err != nil {
+		return fmt.Errorf("failed to create tempNS: %v", err)
+	}
+	defer tempNS.Close()
+
+	var contDev netlink.Link
+
+	// Restore original up state in case of error
+	// This must be done in the containerNS as moving
+	// device between namespaces sets the link down
+	defer func() {
```

Comment:
this is called only from CMDDel, there is no going back, i dont think we need this logic.